src/test/java/org/opensearch/jobscheduler/TestHelpers.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.jobscheduler;
+
+import org.opensearch.common.bytes.BytesReference;
+import org.opensearch.common.xcontent.XContentBuilder;
+import org.opensearch.common.xcontent.XContentType;
+
+import java.io.IOException;
+
+public class TestHelpers {
+
+    public static XContentBuilder builder() throws IOException {
```

Comment:
probably want "xContent" in the method name.

Reply:
ACK